### DISCUSSION TITLE
Does the package will stop working after the new reddit api changes in July 1, 2023 ?

### DISCUSSION THREAD

**Discussion by MohabElkholy** [arifszn/reddit-image-fetcher#24] [Q&A] [answered]
So reddit is now charging 0.24$ per 1000 request what will happen to the package ?

**Comment by arifszn** [comment, marked as answer]
Most probably this package will be obsolete.

**Comment by arifszn** [comment]
Looks like the package is still functional. @MohabElkholy, do you know about the updated changes on the Reddit API? Also please be noted, the package does not need any authentication, so I am not sure if the API changes will affect the package.
